src/components/common/__snapshots__/EdgeCreateDialog.test.js.snap

```diff
@@ -0,0 +1,239 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<EdgeCreateDialog /> Should render correctly with no properties 1`] = `
+<ContextProvider
+  value={
+    Object {
+      "defaultFormats": Object {},
```

Comment:
I guess it also feels a little bit surprising that most of this snapshot is just the translation strings - which are liable to change often and feel like they aren't the focus of what's being tested